NitroxLauncher/OptionPage.xaml

```diff
@@ -4,40 +4,46 @@
-                <Border  HorizontalAlignment="Left" Height="43" BorderBrush="#33FFFFFF" Background="#9928292C" BorderThickness="1" CornerRadius="4,4,4,4" Margin="0,4,0,0">
-                    <TextBox FontFamily="{StaticResource Roboto}" Width="427" Height="43" FontSize="16" MaxLines="1" Foreground="White" TextAlignment="Left" Padding="9" Background="{x:Null}" BorderBrush="{x:Null}" Text="{Binding PathToSubnautica, RelativeSource={RelativeSource FindAncestor, AncestorType=Page}}" SelectionOpacity="-1" ToolTip="Subnautica install location"/>
+                <TextBlock FontFamily="{StaticResource Roboto}" FontSize="12" Foreground="White" Text="{DynamicResource ResourceKey=NitroxOptionGameDirectory}"/>
+                <Border HorizontalAlignment="Left" Height="43" BorderBrush="#33FFFFFF" Background="#9928292C" BorderThickness="1" CornerRadius="4,4,4,4" Margin="0,4,0,0">
+                    <TextBox IsReadOnly="True" FontFamily="{StaticResource Roboto}" Width="427" Height="43" FontSize="16" MaxLines="1" Foreground="White" TextAlignment="Left" Padding="9" Background="{x:Null}" BorderBrush="{x:Null}" Text="{Binding PathToSubnautica, Mode=OneWay}" SelectionOpacity="-1" ToolTip="{DynamicResource ResourceKey=NitroxOptionGameDirectoryTooltip}" />
                 </Border>
             </StackPanel>
+
             <StackPanel Grid.Row="3" Margin="0,18,0,0">
-                <Button Style="{StaticResource BTNOPTIONS}" Background="#19FFFFFF"  Width="147" Height="36" HorizontalAlignment="Left" BorderBrush="#66FFFFFF" Click="ChangeOptions_Click" ToolTip="Point the launcher to your games install location">
-                    <TextBlock FontFamily="{StaticResource Roboto}" Padding="0,1,0,0" FontSize="16" Foreground="White">Change</TextBlock>
+                <Button Style="{StaticResource BTNOPTIONS}" Background="#19FFFFFF"  Width="147" Height="36" HorizontalAlignment="Left" BorderBrush="#66FFFFFF" Click="ChangeOptions_Click" ToolTip="{DynamicResource ResourceKey=NitroxOptionSubmitTooltip}">
+                    <TextBlock FontFamily="{StaticResource Roboto}" Padding="0,1,0,0" FontSize="16" Foreground="White" Text="{DynamicResource ResourceKey=NitroxOptionSubmit}" />
                 </Button>
             </StackPanel>
+
+            <StackPanel Grid.Row="4" Margin="0,30,0,0">
+                <TextBlock FontFamily="{StaticResource Roboto}" FontSize="12" Foreground="White" Text="{DynamicResource ResourceKey=NitroxOptionLanguage}"/>
+                <Border HorizontalAlignment="Left" Height="43" BorderBrush="#33FFFFFF" Background="#9928292C" BorderThickness="1" CornerRadius="4,4,4,4" Margin="0,4,0,0" ToolTip="{DynamicResource ResourceKey=NitroxOptionLanguageTooltip}">
+                    <TextBox IsReadOnly="True" FontFamily="{StaticResource Roboto}" Width="427" Height="43" FontSize="16" MaxLines="1" Foreground="White" TextAlignment="Left" Padding="9" Background="{x:Null}" BorderBrush="{x:Null}" Text="{Binding Lang, Mode=OneWay}" SelectionOpacity="-1"/>
```

Comment:
Readonly for the chosen language? With no ability to change the language at runtime I don't see a reason to provide this `TextBox` in the first place.